test/jdk/java/foreign/TestIllegalLink.java

```diff
@@ -111,52 +111,52 @@ public static Object[][] types() {
         List<Object[]> cases = new ArrayList<>(Arrays.asList(new Object[][]{
             {
                     FunctionDescriptor.of(MemoryLayout.sequenceLayout(2, C_INT)),
-                    "Unsupported layout: [2:i32]"
+                    "Unsupported layout: [2:i4]"
             },
             {
                     FunctionDescriptor.ofVoid(MemoryLayout.sequenceLayout(2, C_INT)),
-                    "Unsupported layout: [2:i32]"
+                    "Unsupported layout: [2:i4]"
             },
             {
-                    FunctionDescriptor.ofVoid(C_INT.withBitAlignment(16)),
+                    FunctionDescriptor.ofVoid(C_INT.withByteAlignment(2)),
                     "Layout bit alignment must be natural alignment"
```

Comment:
```suggestion
                    "Layout byte alignment must be natural alignment"
```
And for all the other occurrences.